src/classes/Blockchain.ts

```diff
@@ -27,7 +27,7 @@ export class Blockchain extends NativeLoader {
       const { url, retry, stopGap, timeout } = config as BlockchainElectrumConfig;
       this.id = await this._bdk.initElectrumBlockchain(url, retry, stopGap, timeout);
     } else if (BlockChainNames.Esplora === blockchainName) {
-      const { url, proxy, concurrency, stopGap, timeout } = config as BlockchainEsploraConfig;
+      const { url, proxy, concurrency, timeout, stopGap } = config as BlockchainEsploraConfig;
```

Comment:
According to the ffi API docs, the `stopGap` does come before the `timeout` for Esplora

https://bitcoindevkit.org/android/bdk-android/org.bitcoindevkit/-esplora-config/-esplora-config.html?query=fun%20EsploraConfig(baseUrl:%20String,%20proxy:%20String?,%20concurrency:%20UByte?,%20stopGap:%20ULong,%20timeout:%20ULong?)